### ISSUE TITLE
Ensure we can generate panan-like forcing

### ISSUE THREAD

**Issue by angus-g** [COSIMA/mom6-regional-scripts#3]
The input generation scripts were lifted out of the EAC configuration, which is an isolated domain with four non-periodic boundaries. Among the more general requirements, we should be able to recreate the https://github.com/cosima/mom6-panan forcing, which is just a single (northern) boundary, but with zonal periodicity.

**Comment by ashjbarnes** [comment]
This was tested by Adele, but I think the Northern boundary for PanAn was far too big to be handled by a notebook. So our tools might work (needs to be tested) but the computational requirement for processing and remapping so much data might require a separate PBS job. Once users are happy using the demo notebooks though they would hopefully be confident to write their own scalable python script using the package

**Comment by ashjbarnes** [comment]
Update: The package can now handle the regridding of larger datasets thanks to update to xESMF. I don't thing that the regridding part of our package will be an issue, but we'll somehow need to tell MOM6 that longitude wraps around the whole globe. Perhaps this can be done with a MOM_override file? Or is this done via the metadata of the netcdf files themselves?

@adele-morrison , assiming you have all your netcdf files correctly regridded, do you know how to "turn on" panan forcing?

**Comment by navidcy** [comment]
Should we revisit this?

**Comment by ashjbarnes** [comment]
Yeah this could be another example notebook? Demonstrate setting things up with a single northern boundary, and you'd also need to add `reentrant_x=True` or something in the MOM_override/input
